html/elements/link.json

```diff
@@ -1150,35 +1150,11 @@
                         "value_to_set": "true"
                       }
                     ]
-                  },
-                  {
-                    "partial_implementation": true,
```

Comment:
If this was removed by the script there's a bug, this isn't flag data at all.

Reply:
This one was a manual change; it's not particularly useful information to record that one single release had support in the past.

Reply:
I agree it's not particularly useful, but still more useful than the flag data that we're still keeping.

Maybe we could have a guideline about removing failed attempts to ship that only lasted one release far in the past, but I think I'd still prefer to keep the data, since the collector could potentially re-discover this and then we'd have to investigate.

Reply:
Oh, I missed that you also removed the same thing for Firefox Android below. I have the same feedback there &ndash; let's remove the flag data only.